slick-testkit/src/codegen/scala/slick/test/codegen/GenerateRoundtripSources.scala

```diff
@@ -136,7 +136,7 @@ class Tables(val profile: JdbcProfile){
     def String = column[String]("String",O.Default("someDefaultString"),O.Length(254))
     def java_sql_Date = column[java.sql.Date]("java_sql_Date")
     def java_sql_Time = column[java.sql.Time]("java_sql_Time")
-    def java_sql_Timestamp = column[java.sql.Timestamp]("java_sql_Timestamp")
+    def java_sql_Timestamp = column[java.sql.Timestamp]("java_sql_Timestamp" ,  O.Default.Now )
```

Comment:
would be good if we could also write a test that checks that this in indeed working correctly and producing a recent time value